mail/views.py

```diff
@@ -73,6 +74,26 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
         )
 
 
+class AutomaticEmailView(GenericAPIView):
```

Comment:
well... you will need to switch class... better to leave `GenericAPIView` if you plan to modify it